File: PerkalianMatriks.java
import java.util.Scanner;
class PerkalianMatriks {
    public static void main(String[] args) {
        Scanner input = new Scanner (System.in);
        int[][] matriks1;
        int[][] matriks2;
        int[][] hasil;
        int baris;
        int kolom;
        char pilih;
        do{
            System.out.println("=======PROGRAM PERKALIAN MATRIKS=======");
            System.out.print("Masukkan banyak baris matriks : ");
            baris = input.nextInt();
            System.out.print("Masukkan banyak kolom matriks : ");
            kolom = input.nextInt();
            System.out.println();

            matriks1 = new int[baris][kolom];
            matriks2 = new int[baris][kolom];
            hasil = new int[baris][kolom];

            System.out.println("Masukkan elemen matriks 1");
            for(int x=0; x<baris; x++){
                for(int y=0; y<kolom; y++){
                    System.out.print("Matriks1 ["+x+"]["+y+"] = ");
                    int isi = input.nextInt();
                    matriks1[x][y] = isi;
                }
            }
            System.out.println("\n");

            System.out.println("Masukkan elemen matriks 2");
            for(int x=0; x<baris; x++){
                for(int y=0; y<kolom; y++){
                    System.out.print("Matriks2 ["+x+"]["+y+"] = ");
                    int isi = input.nextInt();
                    matriks2[x][y] = isi;
                }
            }
            System.out.println("Hasil perkalian matriks: ");
            for(int x = 0; x<baris; x++){
                for(int y = 0; y<kolom; y++){
                    hasil[x][y] = matriks1[x][y] * matriks2[x][y];
                    System.out.print(hasil[x][y]+"  ");
                }
            }
            System.out.println();
            System.out.println("=======================================");
            System.out.print("Apakah anda ingin mengulang program Y/N : ");
            pilih = input.next().charAt(0);

            if(pilih == 'N'){
                System.out.println("Anda telah keluar dari program");
            }
        }while(pilih == 'Y');
    }
}

File: PerkalianMatriks.cpp
#include <iostream>
using namespace std;
int main() {
    int x, y, baris, kolom, matriks1[20][20], matriks2[20][20], hasil[20][20];
    char pilih;
    do{
    cout<<"=======PROGRAM PERKALIAN MATRIKS=======\n"<<endl;
  	cout<<"Masukkan banyak baris matriks: ";cin>>baris;
  	cout<<"Masukkan banyak kolom matriks: ";cin>>kolom;
	cout<<"Masukkan elemen matrix 1: \n";
    for(x = 0; x < baris; x++){
    	for(y = 0; y < kolom; y++){
        	cin >> matriks1[x][y];
    	}
   }
    cout<<"Masukkan elemen matrix 2: \n";
    for(x = 0; x < baris; x++){
    	for(y = 0; y < kolom; y++){
     	    cin>>matriks2[x][y];
  		}
   }
    cout<<"hasil perkalian matrix: \n";
  	for(x = 0; x < baris; x++){
    	for(y = 0; y < kolom; y++){
      		hasil[x][y] = matriks1[x][y] * matriks2[x][y];
      		cout << hasil[x][y] << "\t";
    		}
    	cout<<endl;
  	}
  	cout<<"\n======================================="<<endl;
  	cout <<"\n\nApakah anda ingin Mengulang Program Y/N : ";
    cin >> pilih;
    cout <<"\n";
    if (pilih=='N'){
    	cout<<"Anda telah keluar dari program";
	}
    
}while (pilih =='Y');
    
   return 0;
}
